WalletWasabi.Gui/MainWindow.xaml.cs

```diff
@@ -78,74 +87,114 @@ private async void MainWindow_ClosingAsync(object? sender, CancelEventArgs e)
 			}
 		}
 
-		private async Task ClosingAsync()
+		/// <summary>
+		/// This will try to close the main window of the application.
+		/// </summary>
+		/// <param name="tryToDequeue">If true then the user will get a dialog where a status of the dequing process is shown and the window will be kept open until it successfully finishes. If false then the dequeing process will be handled later at the disposal of the business logic.</param>
```

Comment:
```suggestion
		/// <param name="tryToDequeue">If true then the user will get a dialog where a status of the dequeuing process is shown and the window will be kept open until it successfully finishes. If false then the dequeuing process will be handled later at the disposal of the business logic.</param>
```

Very weird spelling. Leaving up to consideration. Not native speaker. Not sure.

PS: https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater - quite nice MSVS extension

Reply:
https://en.wiktionary.org/wiki/dequeuing

Reply:
I find this easier to read:

```
		/// <param name="tryToDequeue">
		/// If true then the user will get a dialog where a status of the dequeuing process is shown and the window will be kept open until it successfully finishes.
		/// If false then the dequeuing process will be handled later at the disposal of the business logic.
		/// </param>
```

Reply:
ACK